adaptive_scheduler/scheduler_input.py

```diff
@@ -46,15 +64,12 @@ def __init__(self, dry_run=False, run_once=False, no_weather=False,
         self.pickle = pickle
         self.save_output = save_output
         self.request_logs = request_logs
-        self.rr_run_time = rr_run_time
```

Comment:
These instance vars that were removed were just never used?

Reply:
they were duplicates to two instance variables that contain the same values. There was one place this one was used vs. the duplicate but I changed them all to use the one variable. I'm not quite sure how this happened... maybe a bad merge a long time ago? Or I just added them twice way back when...

Reply:
Ah yeah sounds good